code/modules/chemistry/Reagents-FoodDrink.dm

```diff
@@ -3759,3 +3759,77 @@ datum
 			alch_strength = 0.4
 			description = "A warm, late night drink, usually enjoyed during long winter nights."
 			reagent_state = LIQUID
+
+		fooddrink/alcoholic/grenadine
+			name = "Grenadine"
+			id = "grenadine"
+			fluid_r = 234
+			fluid_g = 19
+			fluid_b = 19
+			description = "A sticky, sweet and tart non-alcoholic bar syrup, used in cocktails for it's distinct bright red colour."
+			reagent_state = LIQUID
+
+		fooddrink/pinklemonade
+			name = "Pink lemonade"
+			id = "pinklemonade"
+			fluid_r = 253
+			fluid_g = 230
+			fluid_b = 237
+			description = "A popular twist on cloudy lemonade, this soft drink has been dyed pink. How colourful."
+			reagent_state = LIQUID
+
+		fooddrink/alcoholic/duckfart
+			name = "Duck Fart"
+			id = "duckfart"
+			fluid_r = 253
+			fluid_g = 245
+			fluid_b = 230
+			alch_strength = 0.6
+			description = "An eccentric 'trio cocktail', in which the 3 ingredients have been layed on top on another."
+			reagent_state = LIQUID
+
+		fooddrink/alcoholic/philcollins
+			name = "Phil Collins"
+			id = "philcollins"
+			fluid_r = 240
+			fluid_g = 248
+			fluid_b = 255
+			alch_strength = 0.3
+			description = "A variation on a well known drink, paying tribute to a well known drummer."
+			reagent_state = LIQUID
+
+		fooddrink/alcoholic/spicedrum
+			name = "Spiced rum"
+			id = "spicedrum"
+			fluid_r = 205
+			fluid_g = 149
+			fluid_b = 12
+			alch_strength = 0.6
+			description = "An egregious and disgusting misinterpretation of some perfectly good rum."
+			reagent_state = LIQUID
+
+		fooddrink/alcoholic/beesknees
+			name = "Bee's Knees"
+			id = "beesknees"
+			fluid_r = 255
+			fluid_g = 236
+			fluid_b = 139
+			alch_strength = 0.3
+			description = "A cocktail from the prohibition era, named after a popular expression."
+			reagent_state = LIQUID
+
+		fooddrink/alcoholic/methacola
+			name = "Methacola"
+			id = "methacola"
+			fluid_r = 173
+			fluid_g = 124
+			fluid_b = 76
+			description = "A potent cocktail of amphetamines, caffeine and corn syrup. If this doesn't kill you then I don't know what will."
+			reagent_state = LIQUID
+
+			on_mob_life(var/mob/M, var/mult = 1)
+				if(!M) M = holder.my_atom
+				if(prob(10))
+					M.reagents.add_reagent("methamphetamine", rand(1,10) * mult)
```

Comment:
Sanity check: intent here is basically to have it decay into the meth you made it with? Seems like a way to keep you hopped up on meth with a much lower overdose chance.

Reply:
yea might be an issue. I'll mess around with the values a bit

Reply:
I think as long as there's a decent loss in efficiency (i.e. amount of meth you get out) this is *probably* fine. With a depletion rate (amount per tick, is multiplied by `mult` so that variable can be cancelled out) of 0.4 (default), this is adding in 1-10 (5.5 average) meth 10% of the time, or 0.55 meth per tick (on average), so you actually get *more* meth out of this than you put in. Totally possible that my maths/understanding of the system is wrong.

The depletion rate of meth is 0.6, and an overdose threshold of 20, so in order for this to not be a "safe" (quotes, because it still does brain damage without the overdose) way to just leak a little meth into your system without any real risk of accidentally spiking the overdose (avoiding discussing details of the overdose effects intentionally, please do the same), I would consider upping the depletion rate to at least 0.6 (to be comparable to meth), if not a little higher.

As it stands, with meth giving a 50% chance of brain damage per tick, this reagent is actually a pretty decent poison as low meth effects are probably viewed as beneficial. Chugging 50u of this has a reasonable chance of giving you a fatal dose of meth over the time it's in your system, due to the high variability of the amount added meaning a few unlucky rolls can spike the amount in you.

Reply:
i picked the same formula that the gin and sonic so as not to reinvent the wheel and go with something that works